Sources/Customization/DefaultEventDispatcher.swift

```diff
@@ -256,12 +254,12 @@ open class DefaultEventDispatcher: BackgroundingCallbacks, OPTEventDispatcher {
 extension DefaultEventDispatcher {
     
     func addProjectChangeNotificationObservers() {
-        observerProjectId = NotificationCenter.default.addObserver(forName: .didReceiveOptimizelyProjectIdChange, object: nil, queue: nil) { [weak self] (notif) in
+        observerProjectId = NotificationCenter.default.addObserver(forName: .didReceiveOptimizelyProjectIdChange, object: nil, queue: nil) { [weak self] (_) in
```

Comment:
I'd like to discuss this briefly.  Working on large projects with more than 3 developers, we found using notification center very hard for debugging.  We ended up using the delegate pattern instead.  Thoughts?